Assignment 2 - Menu/src/assignment/pkg2/menu/Assignment2Menu.java

```diff
@@ -0,0 +1,90 @@
+/*
+Dylan Tkach
+Java Programming
+CPT-163-27
+ */
+package assignment.pkg2.menu;
+
+import java.util.Scanner;
+
+public class Assignment2Menu {
+
+    public static void main(String[] args) {
+    
+        Scanner keyboard = new Scanner(System.in);
+        
+        int MenuSelection;
+        
+        int charW = 'W';
+        int charA = 'A';
+        int charR = 'R';
+        int charH = 'H';
+        // A declared.
+        int charM = 'M';
+        // M declared.
+        int charE = 'E';
+        // R declared.
+        
```

Comment:
I personally did the declarations and calculations for this menu option in that part of the switch statement. There is nothing wrong with how you did it. I just think it looked cleaner and easier to follow that way for me.